src/app/Client/GeneratePrompt.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import { generate } from "random-words";
+import { useContext, useEffect } from "react";
+import { ACTIONS, GENRES, LINES, NAMES } from "../Helpers/constants";
+import { PromptContext } from "./Contexts/PromptFeatures";
+
+const GeneratePrompt = () => {
+  const {
+    requiredWord, setRequiredWord,
+    genre, setGenre,
+    action, setAction,
+    name, setName,
+    line, setLine,
+  } = useContext(PromptContext);
+  
+  useEffect(() => {
+    const ws = generate(5);
+    setRequiredWord(ws[4]); // theres no reason to use the last word, this is just a hack to ensure the type is string rather than string | string[] 
```

Comment:
im not thrilled about this.  it would be nice to be able to expect the correct type from generate() with no arguments.